lib/requestFactory/index.js

```diff
@@ -124,6 +115,41 @@ class RequestFactory {
     })
   }
 
+  async getRequestsByBucket(bucket) {
+    const logs = await this.getRequestCreatedLogs({
+      bucket,
+    })
+    const requests = []
+    logs.forEach((log) => {
+      requests.push(log.args.request)
```

Comment:
New scheme for events is that they carry all necessary info, which effectively reduces need to use fillData(). Here we are stripping the info and reduce to address only. I would recommend to return whole events will all scheduling data.